src/Collection/Collection.router.spec.ts

```diff
@@ -904,7 +904,7 @@ describe('Collection router', () => {
         q = 'collection name 1'
         totalCollectionsFromDb = 1
         baseUrl = '/collections'
-        url = `${baseUrl}?limit=${limit}&page=${page}&assignee=${assignee}&status=${status}&sort=${sort}&is_published=${isPublished}&q=${q}`
+        url = `${baseUrl}?limit=${limit}&page=${page}&assignee=${assignee}&status=${status}&sort=${sort}&isPublished=${isPublished}&q=${q}`
```

Comment:
Modify the param `is_published` to `isPublished` because the front end sends it in CamelCase format

Reply:
What do you think if we change the frontend to send it snake case? I'd advocate for that since we have some other query params already as snake case, so we keep using the same standard.